chart/samples/postgresql.appcat.vshn.io_postgresqlstandaloneconfig.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: postgresql.appcat.vshn.io/v1alpha1
```

Comment:
Do we really need another CR for specifying the amount resources? Do we expect to add more data here? Where do we save the Helm Values for the chart?

Reply:
We need some way where we can limit the resource requests per instance.
For example, on Cloudscale we might want to limit the instances between 5 GB and 500 GB, while on Exoscale there could be different sizes. The same goes for Memory.

And that is exactly what this CR is used for: Like a configmap that configures the operator. The operator then uses this config (either at startup or at runtime, tbd) to then configure webhook server for example.

And the Helm values are also different per platform (for example different storage classes) so we need a way that isn't hardcoded (unless we want to hardcode everything in the operator, but that means a new operator version for every little config change, not sure if that's DX friendly)

Reply:
Maybe that CRD could be renamed, because right now it's a bit ambiguous and can confuse people...

Reply:
Let's discuss this further on the PR level